data/lakshay451.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "Lakshay Joshi",
+  "bio": "Hi, I am a final year undergrad @DTU (DCE). I love Web development and contributing to Open source projects.",
+  "links": [
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/lakshay-joshi-b9298b201/",
+      "icon": "FaLindein"
+    }
```

Please add the missing comma here! 

```suggestion
    },
```